terraform/fullnode/aws/backup.tf

```diff
@@ -0,0 +1,67 @@
+resource "random_id" "backup-bucket" {
+  byte_length = 4
+}
+
+resource "aws_s3_bucket" "backup" {
```

Comment:
we may want to throw cloudfront in front of s3 to help absorb some of the egress cost , but that can come in a separate PR